src/download/linux.md

```diff
@@ -139,32 +139,13 @@ sudo dnf install prismlauncher
 
 ## <img src="https://www.vectorlogo.zone/logos/debian/debian-icon.svg" alt="Debian Logo" height="20" /> Debian / <img src="https://www.vectorlogo.zone/logos/ubuntu/ubuntu-icon.svg" alt="Ubuntu Logo" height="20" /> Ubuntu (x86_64, ARM64)
 
-Several packages are available through [makedeb](https://makedeb.org/)'s [MPR](https://mpr.makedeb.org/)
-
-[![MPR: prismlauncher](https://img.shields.io/badge/mpr-prismlauncher-orange)](https://mpr.makedeb.org/packages/prismlauncher)
-[![MPR: prismlauncher-git](https://img.shields.io/badge/mpr-prismlauncher--git-orange)](https://mpr.makedeb.org/packages/prismlauncher-git)
-
-### Installation using Prebuilt-MPR (recommended)
-
-[Add the Prebuilt-MPR repository](https://docs.makedeb.org/prebuilt-mpr/getting-started/), then run:
-
-```bash
-sudo apt install prismlauncher
-```
-
-Prebuilt MPR supports Debian 11/12, and Ubuntu 20.04, 22.04, 24.04.
-
-### Installing with mist, the MPR CLI
-
-Install `mist` by following the [upstream instructions](https://docs.makedeb.org/using-the-mpr/mist-the-mpr-cli/#installing-mist).
-
-Packages installed via `mist` will be compiled from source
+Packages are made available for Ubuntu, Debian, and Linux Mint through the unofficial [*Prism Launcher for Debian* repository](https://github.com/Prism-Launcher-for-Debian/repo).
```

Comment:
```suggestion
Packages are made available for Ubuntu, Debian, and Linux Mint through the [*Prism Launcher for Debian* repository](https://github.com/Prism-Launcher-for-Debian/repo).
```

We don't use unofficial anywhere else since the entire section is marked as community packages so shouldnt use it here, otherwise lgtm